api-schema/src/main/java/org/stellar/anchor/api/sep/sep6/Refunds.java

```diff
@@ -0,0 +1,18 @@
+package org.stellar.anchor.api.sep.sep6;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Refunds {
```

Comment:
We have the class: `package org.stellar.anchor.api.shared.Refunds`. Can we share?